test/Spago/Build/Monorepo.purs

```diff
@@ -246,3 +252,110 @@ spec = Spec.describe "monorepo" do
 
       spago [ "build" ] >>= shouldBeSuccess
       spago [ "build", "--pedantic-packages" ] >>= shouldBeFailureErr (fixture "monorepo/pedantic-cross-package-imports/expected-stderr.txt")
+
+  Spec.it "#1208: clones a monorepo only once, even if multiple packages from it are needed" \{ spago, fixture, testCwd } -> do
+    -- A local file system Git repo to use as a remote for Spago to clone from
+    let createLibraryRepo = do
+          let libRepo = Path.concat [ Paths.paths.temp, "spago-1208" ]
```

Comment:
This is another path length saving measure: the repo is always kept in `$TEMP/spago-1208` rather than a unique directory with a mile-long name.

Reply:
Makes sense 👍